### PR TITLE
Consolidated Request for feedback

### PR DESCRIPTION
**Pull request by dbojicic**
Consolidated request for feedback notes - organised by the type of change 

- AU Core Encounter
- AU Core Immunization
- AU Core Organization
- AU Core Practitioner 
- AU Core PractitionerRole

### CODE REVIEW

File: input/intro-notes/StructureDefinition-au-core-condition-intro.md [hunk 1]
```diff
@@ -29,5 +29,4 @@ Conformance in reverse is not guaranteed, i.e. a resource conforming to [Interna
 
 <p class="request-for-feedback">Specific feedback is sought on how to do 'free text' medical history i.e. are these always Condition?<br/>Please comment on HL7 Jira <a href="https://jira.hl7.org/browse/FHIR-43846">FHIR-43846</a>.</p>
 
-<p class="request-for-feedback">Specific feedback is sought on how a diagnosis type (e.g. Comorbidity, Complication) would be represented in AU Core Condition.<br/>Please comment on HL7 Jira <a href="https://jira.hl7.org/browse/FHIR-43847">FHIR-43847</a>.</p>
-
+<p class="request-for-feedback">Specific feedback is sought on how a diagnosis type (e.g. Comorbidity, Complication) would be represented in AU Core Condition.<br/>Please comment on HL7 Jira <a href="https://jira.hl7.org/browse/FHIR-43847">FHIR-43847</a>.</p>
```
Comment (dt-r) [inline input/intro-notes/StructureDefinition-au-core-condition-intro.md:32]: What is this diff?

File: input/intro-notes/StructureDefinition-au-core-encounter-intro.md [hunk 2]
```diff
@@ -23,13 +23,12 @@ Conformance in reverse is not guaranteed, i.e. a resource conforming to [US Core
...
-<p class="request-for-feedback">Specific feedback is sought on the proposal to remove Must Support from Encounter.serviceType, as its inclusion in AU Core may be premature without clear overall support from existing systems. Furthermore, the associated preferred binding is unlikely to be supported by existing systems, suggesting that support for free text or local codes may be the most feasible approach at this point.<br/>Please comment on HL7 Jira <a href="https://jira.hl7.org/browse/FHIR-44581">FHIR-44581</a>.</p>
-
-<p class="request-for-feedback">Specific feedback is sought on the proposal to remove Must Support from Encounter.reasonReference, as its inclusion in AU Core may be premature without clear overall support from existing systems.<br/>Please comment on HL7 Jira <a href="https://jira.hl7.org/browse/FHIR-44582">FHIR-44582</a>.</p>
-
-
-
-
-
+<div class="request-for-feedback">
+    <p>Specific feedback is sought on a series of proposals to remove Must Support from the following elements:
```
Comment (dt-r) [inline input/intro-notes/StructureDefinition-au-core-encounter-intro.md:27]: remove the following words across the files "a series of"

File: input/intro-notes/StructureDefinition-au-core-immunization-intro.md [hunk 3]
```diff
@@ -15,22 +15,24 @@ A resource conforming to this profile is conformant to:
 
 Conformance in reverse is not guaranteed, i.e. a resource conforming to [International Patient Access](https://build.fhir.org/ig/HL7/fhir-ipa), [International Patient Summary](http://build.fhir.org/ig/HL7/fhir-ips), or [US Core](http://hl7.org/fhir/us/core) **MAY NOT** conform to AU Core.
 
-
-### Profile specific implementation guidance
```
Comment (dt-r) [inline input/intro-notes/StructureDefinition-au-core-immunization-intro.md:19]: Restore the deleted heading and implementation guidance

**Review by dt-r — CHANGES_REQUESTED** [review]
See comments.

File: input/intro-notes/StructureDefinition-au-core-practitioner-intro.md [hunk 4]
```diff
@@ -23,10 +23,13 @@ Conformance in reverse is not guaranteed, i.e. a resource conforming to [Interna
...
-
-<p class="request-for-feedback">Specific feedback is sought on the proposal to remove Must Support from Practitioner.communication, as its inclusion in AU Core may be premature without clear support from existing systems.<br/>Please comment on HL7 Jira <a href="https://jira.hl7.org/browse/FHIR-44588">FHIR-44588</a>.</p>
+<div class="request-for-feedback">
+    <p>Specific feedback is sought on a series of proposals to remove Must Support from the following elements:
+    <ul>
+        <li>Practitioner.qualification.period and Practitioner.qualification.issuer, as their inclusion in AU Core may be premature without clear support from existing systems. Additionally, issuer can be derived from the AHPRA identifier assigner.<br/>Please comment on HL7 Jira <a href="https://jira.hl7.org/browse/FHIR-44587">FHIR-44587</a>.</li>
+        <li>Specific feedback is sought on the proposal to remove Must Support from Practitioner.communication, as its inclusion in AU Core may be premature without clear support from existing systems.<br/>Please comment on HL7 Jira <a href="https://jira.hl7.org/browse/FHIR-44588">FHIR-44588</a>.</li>
```
Comment (dt-r) [inline input/intro-notes/StructureDefinition-au-core-practitioner-intro.md:30]: Remove the words "Specific feedback is sought on the proposal to remove Must Support from " for the point on Practitioner.communication

**Comment by mjosborne1** [comment]
CodeSystem [http://terminology.hl7.org/CodeSystem/allergyintolerance-verification](https://hl7.org/fhir/R4/codesystem-allergyintolerance-verification.html) is missing from the terminology page under "Code systems published in FHIR". AllergyIntolerance.verificationStatus is Must Support.

File: input/intro-notes/StructureDefinition-au-core-encounter-intro.md (hunk 2 above)
Comment (dbojicic) [reply input/intro-notes/StructureDefinition-au-core-encounter-intro.md:27]: Done

File: input/intro-notes/StructureDefinition-au-core-immunization-intro.md (hunk 3 above)
Comment (dbojicic) [reply input/intro-notes/StructureDefinition-au-core-immunization-intro.md:19]: Done

File: input/intro-notes/StructureDefinition-au-core-practitioner-intro.md (hunk 4 above)
Comment (dbojicic) [reply input/intro-notes/StructureDefinition-au-core-practitioner-intro.md:30]: Done

File: input/intro-notes/StructureDefinition-au-core-condition-intro.md (hunk 1 above)
Comment (dbojicic) [reply input/intro-notes/StructureDefinition-au-core-condition-intro.md:32]: No diff.

File: input/intro-notes/StructureDefinition-au-core-immunization-intro.md [hunk 5]
```diff
@@ -1,4 +1,4 @@
-### Usage scenarios
+### Usage scenarios,
```
[the remark this answers is not in the document]
Comment (dt-r) [reply input/intro-notes/StructureDefinition-au-core-immunization-intro.md:1]: comma typo still present

**Review by dt-r — CHANGES_REQUESTED** [review]
comma typo still present

File: input/intro-notes/StructureDefinition-au-core-immunization-intro.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (dbojicic) [reply input/intro-notes/StructureDefinition-au-core-immunization-intro.md]: ah :( Removed now.

**Review by dt-r — APPROVED** [review]

**Pull request merged**